eodag/cli.py

```diff
@@ -482,27 +488,70 @@ def download(ctx, **kwargs):
 
     satim_api = EODataAccessGateway(user_conf_file_path=conf_file)
     search_results = satim_api.deserialize(search_result_path)
-    # register downloader
-    for idx, product in enumerate(search_results):
-        if product.downloader is None:
-            auth = product.downloader_auth
-            if auth is None:
-                auth = satim_api._plugins_manager.get_auth_plugin(product.provider)
-            search_results[idx].register_downloader(
-                satim_api._plugins_manager.get_download_plugin(product), auth
+
+    get_quicklooks = kwargs.pop("quicklooks")
+    if get_quicklooks:
+        click.echo(
+            "Flag 'quicklooks' specified, downloading only quicklooks of products"
+        )
+
+        for product in search_results:
+            quicklook_link = product.properties.get("quicklook")
+
+            if not quicklook_link:
+                click.echo(
+                    "Error during download, EO product '{}' does not provide quicklook link".format(
+                        product.properties["title"]
+                    )
+                )
+                continue
+
+            download_plugin = satim_api._plugins_manager.get_download_plugin(product)
+
+            outputs_prefix = (
+                kwargs.get("outputs_prefix", None)
+                or download_plugin.config.outputs_prefix
             )
+            file_ext = os.path.splitext(os.path.basename(quicklook_link))[1]
+            quicklook_file = (
+                sanitize(product.properties["title"]) + "--quicklook" + file_ext
+            )
+            base_dir = outputs_prefix
```

Comment:
```suggestion
```

Reply:
The CLI should behave the same way the Python API does: downloading quicklooks in `{outputs_prefix}/quicklooks/{product_id}` (without extension). This way, the user will know in advance where the quicklook will be downloaded without having to look for the file extension or the file name